terraform/modules/vpc/vpc.tf

```diff
@@ -96,11 +115,15 @@ resource "aws_subnet" "sn" {
   vpc_id            = "${aws_vpc.vpc.id}"
   map_public_ip_on_launch = true
 
-  tags = {
-    Name = "${var.component_name}-${var.deployment_stage}-${local.az_names[count.index]}"
-    component = "${var.component_name}"
-    deployment_stage = "${var.deployment_stage}"
-  }
+  tags = "${merge(
+    var.default_tags,
+    map(
+      "Name","${var.component_name}-${var.deployment_stage}-${local.az_names[count.index]}",
+      "Env","${var.deployment_stage}",
+      "component","${var.component_name}",
+      "deployment_stage","${var.deployment_stage}"
+    )
+  )}"
 }
 
 resource "aws_route_table_association" "a" {
```

Comment:
there's no tag support for this resource in tf

Reply:
I think I tried adding it and it's throwing an error that it doesn't recognise the tags keyword

Reply:
Yes, actually this isn't being tagged in the terraform config.